R/pkg/R/DataFrame.R

```diff
@@ -1859,7 +1859,7 @@ setMethod("where",
 #' @param colnames A character vector of column names.
```

Comment:
That sounds good. Would also be good to include `if col contains a character vector then additional column names are ignored` (since thats what the implementation is doing from what I see ?)